test/e2e/services.go

```diff
@@ -73,6 +105,64 @@ func startServicesForLogs(t *testing.T, e e2e.Environment) (
 	return loki.InternalEndpoint("http"), loki.Endpoint("http")
 }
 
+func startServicesForTraces(t *testing.T, e e2e.Environment) (otlpGRPCEndpoint string, jaegerHttpEndpoint string) {
+	jaeger := e.Runnable("jaeger").
+		WithPorts(
+			map[string]int{
+				"jaeger.grpc": 14250, // Receives traces
+				"grpc.query":  16685, // Query
+				"http.query":  16686, // Query
+			}).
+		Init(e2e.StartOptions{Image: "jaegertracing/all-in-one:1.31"})
+
+	// Warn if a YAML change introduced a tab character
+	if strings.ContainsRune(otelConfig, '\t') {
+		t.Errorf("Tab in the YAML")
+	}
+
+	config := strings.Replace(otelConfig,
+		"{{JAEGER_GRPC_ENDPOINT}}",
+		jaeger.InternalEndpoint("jaeger.grpc"), -1)
+
+	otelFile, err := ioutil.TempFile(e.SharedDir(), "collector.yaml")
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+
+	defer os.Remove(otelFile.Name())
+
+	_, err = otelFile.Write([]byte(config))
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+
+	// otelFile.Name() will give relative pathname to temp file.  Docker will complain with
+	// "If you intended to pass a host directory, use absolute path.""
+	otelFileName, err := filepath.Abs(otelFile.Name())
+	if err != nil {
+		t.Fatalf("unexpected Abs() error: %s", err)
+	}
+
+	otel := e.Runnable("otel-collector").
+		WithPorts(
+			map[string]int{
+				"grpc": 4317,
+				"http": 4318,
+			}).
+		Init(e2e.StartOptions{
+			Image:   "otel/opentelemetry-collector:0.45.0",
```

Comment:
I think we could move this to a const [here](https://github.com/observatorium/api/blob/ca9ffa0586a31db2ef842713ccfc1363fc783a96/test/e2e/services.go#L21) so is easier to replace in case the image is updated